app/assets/javascripts/preferences/models/preferences.ts

```diff
@@ -0,0 +1,76 @@
+import { IconType } from '@/components/Icon';
```

Comment:
This model is pretty similar to the rest of the models on the shared state, so I think it'd make more sense to keep it in appState too